### ISSUE TITLE
LineLength checker provides incorrect line length

### ISSUE THREAD

**Issue by gdinant** [checkstyle/checkstyle#7146]
/var/tmp $ javac MyClass.java
```java 
public class MyClass {

	private String lineLengthTest = "This is a 120 line length test -------------------------------------------------";

}
```

/var/tmp $ cat config.xml
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
	"-//Puppy Crawl//DTD Check Configuration 1.3//EN"
	"https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">

	<property name="charset" value="UTF-8" />

	<module name="LineLength">
		<property name="max" value="121" />
		<property name="fileExtensions" value="java" />
	</module>

</module>
```
```bash
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.25-all.jar -c config.xml MyClass.java
/var/tmp $ java -jar checkstyle-8.25-all.jar -c config.xml MyClass.java  
Starting audit...
[ERROR] /var/tmp/MyClass.java:3: Line is longer than 121 characters (found 123). [LineLength]
Audit done.
Checkstyle ends with 1 errors.
```

---------------

The actual line length is 120 whereas checkstyle reports 123 and reports a false positive error.

Since LineLength was moved out of the TreeWalker I experience this. Works fine with version 8.23.

It's blocking me from upgrading as it impacts tons of project files.

--------------

**Comment by gdinant** [comment]
I found the defect. Since my file has tabulations, I also had to move the property `tabWidth` to the checker definition and not only in the `TreeWalker`.

```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
	"-//Puppy Crawl//DTD Check Configuration 1.3//EN"
	"https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">

	<property name="charset" value="UTF-8" />

	<property name="tabWidth" value="4" />

	<module name="LineLength">
		<property name="max" value="121" />
		<property name="fileExtensions" value="java" />
	</module>

</module>
```

Closing :)
